data/datalist

```diff
@@ -3,4 +3,5 @@ algdat.p20
 algdat.pfull
 fl25
 fl70
-fl250
\ No newline at end of file
+fl250
+east_tx
```

Comment:
We are probably pushing beyond the maximum data size for packages here. It's okay for now, but before github submission, we should talk about a redistData package. (It would be a good home for a number of datasets that we work with semi-often too)

Reply:
Additionally, I think it needs an entry in data.R, unless something wasn't loading properly above.